src/components/Image.js

```diff
@@ -23,7 +23,7 @@ const addQueryParam = (url, param) => {
 };
 
 export const Image = ({
-  source,
+  source: sourceProp,
```

Comment:
i have difficulties with the naming of `sourceProp`, but i understand the reason.
maybe we could rename the state variable `source` to something else to be able to keep `source` here maybe `imageSource`? or we could use `...props` here like in `ImageWithBadge` and access `props.source` everywhere where `sourceProp` is used now?